src/njoy21/input/THERMR/Card1/Nendf.hpp

```diff
@@ -0,0 +1,29 @@
+struct Nendf {
+  using Value_t = int;
+  static std::string name(){ return "nendf"; }
+  static std::string description(){
+    return
+      "The nendf argument specifies the io unit used internally by the\n"
+      "Fortran routine and the ENDF-format tape to be read in, e.g. setting\n"
+      "nendf to 21 or -21 implies a endf tape named 'tape21' is available in\n"
+      "the working directory. BROADR extracts values for the average number\n"
+      "of neutrons emitted from thermal fission from this tape.\n"
+      "\n"
+      "The signedness of the argument implies the formatting of the tape in\n"
+      "question. Positive argument values indicate formatted input i.e.\n"
+      "ascii text files. Negative values indicate unformatted input i.e.\n"
+      "Fortran-style block binary\n"
+      "\n"
+      "For thermr in particular, the nendf argument provides an endf\n"
+      "file that corresponds to an MF (endf file label) = 7 (thermal \n"
+      "scattering).\n"
+      "\n"
+      "nendf values are restricted to equal either 0, some integer with\n"
```

Comment:
So test problem #1 in the manual has a value of zero, and pg. 174 of the manual states
"The required ENDF tape (nendf) is only used for MF=7 data; it can be set to zero if only free-gas scattering is needed."
Which this makes me think, I should probably set a check to ensure that if nendf == 0, then card2's iin should be equal to 1 (free gas).